### ISSUE TITLE
[terra-image] Missing alternative text in examples gives axe violation

### ISSUE THREAD

**Issue by saket2403** [cerner/terra-core#2723]
## Description 
Some examples in terra-image are missing the values for `alt` prop which gives axe violations.

## Error Report
```
Ensures <img> elements have alternate text or a role of none or presentation
```
